packages/docs/stories/components/tooltip.stories.ts

```diff
@@ -0,0 +1,320 @@
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+import type { Meta, StoryObj } from '@storybook/web-components';
+import { html } from 'lit';
+import '../../../components/src/components/tooltip/tooltip.js';
+import {
+  generateScreenshotStory,
+  generateStoryDescription,
+  storybookDefaults,
+  storybookHelpers,
+  storybookTemplate,
+} from '../../src/helpers/component.js';
+import { generateFigmaPluginObject } from '../../src/helpers/figma.js';
+
+const { args: defaultArgs, argTypes } = storybookDefaults('syn-tooltip');
+const { overrideArgs } = storybookHelpers('syn-tooltip');
+const { generateTemplate } = storybookTemplate('syn-tooltip');
+
+const meta: Meta = {
+  args: overrideArgs([
+    {
+      name: 'content',
+      type: 'attribute',
+      value: 'This is a tooltip',
+    },
+    {
+      name: 'open',
+      type: 'attribute',
+      value: true,
+    },
+    {
+      name: 'default',
+      type: 'slot',
+      value: '<syn-button>Hover me</syn-button>',
+    },
+  ], defaultArgs),
+  argTypes,
+  component: 'syn-tooltip',
+  parameters: {
+    design: generateFigmaPluginObject('12852-79550'),
+    docs: {
+      description: {
+        component: generateStoryDescription('tooltip', 'default'),
+      },
+      story: {
+        height: '200px',
+        // Unfortunately we need the iframes here, otherwise the tooltips placing is broken
+        inline: false,
+      },
+    },
+    layout: 'centered',
+  },
+  title: 'Components/syn-tooltip',
+};
+export default meta;
+
+type Story = StoryObj;
+
+export const Default: Story = {
+  parameters: {
+    controls: {
+      disable: false,
+    },
+    docs: {
+      description: {
+        story: generateStoryDescription('tooltip', 'default'),
+      },
+    },
+  },
+  render: (args: unknown) => generateTemplate({ args }),
+} as Story;
+
+export const Placement: Story = {
+  parameters: {
+    docs: {
+      description: {
+        story: generateStoryDescription('tooltip', 'placement'),
+      },
+      story: {
+        height: '450px',
+      },
+    },
+    layout: 'padded',
+  },
+  render: () => html`
+    <div class="tooltip-placement-example">
+      <div class="tooltip-placement-example-row">
```

Comment:
The issue seems to go away when using `overflow: hidden` on the placement example. However, in this case the container is not wide enough anymore to show the left-start and left-end examples in their "natural" placement. Mind to play around with this a bit?


```suggestion
    <!-- Not those are just values to play with! -->
    <div class="tooltip-placement-example" style="height: 400px; overflow: hidden;">
```

Reply:
This is a problem shoelace also has in its documentation. 
I think the problem appears because the trigger is set on `hover` per default (and I do not change it) but I open all tooltips via the `open` prop.
Therefore i would assume there is no problem with the tooltips themselves.
I spoke with Stefan and we decided we won`t investigate more in it, because it could only be a problem of our documentation.

If you`d like to investigate more, you can have a look at it.

Reply:
I am actually fine with this :)